src/edgepi/tc/tc_constants.py

```diff
@@ -276,7 +276,7 @@ class TCType(Enum):
     TYPE_R = OpCode(0x05, TCAddresses.CR1_W.value, Masks.CR1_LOW_MASK.value)  # type R thermocouple
     TYPE_S = OpCode(0x06, TCAddresses.CR1_W.value, Masks.CR1_LOW_MASK.value)  # type S thermocouple
     TYPE_T = OpCode(0x07, TCAddresses.CR1_W.value, Masks.CR1_LOW_MASK.value)  # type T thermocouple
-
+    TYPE_MASK = 0x07
```

Comment:
Why is this pair different from the rest? Is inconsistency ok in this case?

Reply:
good point and fixed aef1556b20336d5d0c033db5c414ed15cc7184e0